packages/browserslist-config/index.js

```diff
@@ -0,0 +1 @@
+module.exports = ['last 1 Chrome version', 'last 1 Firefox version', 'last 1 Safari version', 'last 1 Edge version'];
```

Comment:
This should be the least-permissive set of focus-trap”: “4.0.2”, “react-dropzone”: “9.0.0”, and “tippy.js”: “5.1.2”. Do you have time to review these dependencies and make a more permissive set?

If you don't have the time, I recommend removing the `isSupportedBrowser` function since it will incorrectly return `false` for older versions.